docs/devel-quick-start.md

```diff
@@ -54,175 +49,73 @@ enough resources:
 
    And make the hook executable:
 
-   ```
+   ```sh
    chmod +x .git/hooks/commit-msg
    ```
 
-1. Create python virtual environment
-
-   The *Ramen* project use python tool to create and provision test
-   environment and run tests. The create a virtual environment including
-   the tools run:
-
-   ```
-   make venv
-   ```
-
-   This create a virtual environment in `~/.venv/ramen` and a symbolic
-   link `venv` for activating the environment. To activate the
-   environment use:
```

Comment:
You cannot delete this part, the environment should be created before you try to use drenv. This is important step that cannot be hidden.

Reply:
Brought this content back.